### DISCUSSION TITLE
Does Drag and Drop for videos supposed to work the same as the Upload link?

### DISCUSSION THREAD

**Discussion by xan-cudo** [immich-app/immich#11502] [Q&A]
Before I logged a bug, I wanted to ask... I have been trying to get videos into Immich. 

- With the Upload link, where I browse my filesystem, it works great. 
- If I just grab one, even the smallest of size, it fails, but with what I think is a Duplicate error. They don't exist in the system already.

I have search, and found hits on MIME issues etc, so I converted some to various formats and they also failed (with a duplicate error) with D&D.

So I thought I'd ask.

Oh, I am running Immich behind nginx, so I bypassed that straight to the http port. Same results.

![Screenshot from 2024-07-31 19-19-31](https://github.com/user-attachments/assets/eb1be65d-ff6d-42ed-8ea5-a460d64e8b36)

**Comment by alextran1502** [comment]
The videos shown in the screenshot looks like a corrupted file?

**Reply by xan-cudo** [reply]
I dont think so. I dont have 500+ corrupt videos. In fact, I can upload to iCloud and they work just fine. I can upload to my Proton drive and they work just fine. I can play them on MPV, Handbrake, VLC on my desktop and they work just fine. I can upload to Immich via the UPLOAD link and they work just fine. But every attempt to upload via D&D gives that exact same result.

But I think what I'm hearing is that is should work, either way, regardless. But it doesn't with my setup. So tomorrow I will try different browsers etc, and see what happens. And the fact that Immich detects as a duplicate is an issue in itself.

I'm happy to provide you samples, or dig thru some logs for you. I simply was reporting an odd issue that I am having,
